lab_5_scrapper/scrapper.py

```diff
@@ -120,45 +252,109 @@ def __init__(self, full_url: str, article_id: int, config: Config) -> None:
         """
         Initializes an instance of the HTMLParser class
         """
-        pass
+        self.full_url = full_url
+        self.article_id = article_id
+        self.config = config
+        self.article = Article(self.full_url, self.article_id)
 
     def _fill_article_with_text(self, article_soup: BeautifulSoup) -> None:
         """
         Finds text of article
         """
-        pass
+        article_content = article_soup.find("div", {'class': "article-body__content"})
+        paragraphs = article_content.find_all("p")
+        self.article.text = ' '.join(paragraph.text.strip() for paragraph in paragraphs)
+        print(self.article.text)
 
     def _fill_article_with_meta_information(self, article_soup: BeautifulSoup) -> None:
         """
         Finds meta information of article
         """
-        pass
+        title = article_soup.find('h1', class_="article-body__title")
+        if title:
```

Comment:
this condition looks like there can be cases when the title field will remain unfilled 
it is unacceptable, title field must always be filled

Reply:
to be pushed